app/controllers/application_controller.rb

```diff
@@ -56,6 +56,6 @@ def current_path(*); end
   def next_path(*); end
 
   def current_app
-    @current_app ||= App.find_or_create_by!(user: current_user)
+    @current_app ||= MbSnapApp.find_or_create_by!(user: current_user)
```

Comment:
Naming: Mb is "Michigan Bridges"? The whole application is for Michigan, so perhaps it's not necessary to specify that here?

Other ideas: `SnapApplication`, `SnapApp`, `MichiganSnapApp`